PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@ was changed. Do not include the issue number in the title. -->
 #### References to other Issues or PRs
 <!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
 format, e.g. "Fixes #1234". See
-https://github.com/blog/1506-closing-issues-via-pull-requests . Please also
+https://github.com/blog/1506-closing-issues-via-pull-requests Please also
 write a comment on that issue linking back to this pull request once it is
 open. -->
```

Comment:
Can't the period go next to the URL?

Reply:
Then how about

```
e.g. "Fixes #1234" (see https://tinyurl.com/auto-closing for more information).
Also, please write...
```

Actually, there is no need to write a comment on the issue because the PR automatically adds a note when a PR references an issue.

Reply:
The advantage of writing a comment is that all participants get an email notification whereas the automatic notes are only seen by someone who explicitly goes to look at the issue.

Reply:
Yeah and also new contributors tend to not see the cross references, so they don't realize that the pull request is being worked on. Unfortunately, people tend to not follow these instructions, so we need some better solutions (like https://github.com/sympy/sympy-bot/issues/72).

Reply:
If it were Markdown rendered, putting the period next to the URL would be fine, because it would separate it from the actual link. But in this, people have to copy-paste the URL, and if you add a period to the end of a URL in your web browser, it will read it as a different URL and give a 404.